PA_Feature_Detecting.md

```diff
@@ -106,14 +106,54 @@ const reportingTimeoutEnabled = navigator.protectedAudience ?
 ```
 
 ## Returning multiple bids from generateBid()
+[Intent to Ship](TBD)
 
 Inside `generateBid()`, if `browserSignals.multiBidLimit` exists then returning
 an array of bids is supported. The value of `browserSignals.multiBidLimit`
 returns the maximum numbers of bids that can be returned, which may be as low as
 1.
 
 ## Component ad subsetting with targetNumAdComponents
+[Intent to Ship](TBD)
 
 Inside `generateBid()`, if `browserSignals.multiBidLimit` exist then
 the `targetNumAdComponents` and `numMandatoryAdComponents` bid fields will be
 considered.
+
+## Cross-origin trusted signals
+[Intent to Ship](TBD)
+
+From context of a web page:
+```
+navigator.protectedAudience && navigator.protectedAudience.queryFeatureSupport(
+    "permitCrossOriginTrustedSignals")
+```
+
+## Getting browser-side detectable features as an object
+Sometimes it's desirable to get status of all features detectable via `queryFeatureSupport` in a
+forward-compatible way. Sufficiently recent versions provide this functionality via
+`queryFeatureSupport('*')`, which returns a dictionary describing state of various features. Since
```

Comment:
a feature request in prebid exists to put the results of this in payloads. Are there any assurances this object won't grow beyond some reasonable size, eg 12 items?

Reply:
I think this list continue to will grow, but, as discussed in a couple of the recent WICG calls, there are various things we can do to limit it's growth.  I think one solution was to pass in a Chrome version, which in turn would make the API only inform you of features that are experimentally enabled beyond what that particular version of Chrome Stable supports by default.